src/encoded/static/components/matrix_target.js

```diff
@@ -0,0 +1,776 @@
+                organismName: 'Homo sapiens',
+                assayTitle: 'TF ChIP-seq',
+                title: 'Homo sapiens | TF ChIP-seq',
+                url: 'replicates.library.biosample.donor.organism.scientific_name=Homo sapiens&assay_title=TF ChIP-seq',
+            }, {
+                organismName: 'Mus musculus',
+                assayTitle: 'Histone ChIP-seq',
+                title: 'Mus musculus | Histone ChIP-seq',
+                url: 'replicates.library.biosample.donor.organism.scientific_name=Mus musculus&assay_title=Histone ChIP-seq',
+            }, {
+                organismName: 'Mus musculus',
+                assayTitle: 'TF ChIP-seq',
+                title: 'Mus musculus | TF ChIP-seq',
+                url: 'replicates.library.biosample.donor.organism.scientific_name=Mus musculus&assay_title=TF ChIP-seq',
+            },
+        ];
+
+        const fontColors = globals.biosampleTypeColors.colorList(this.targetMatrixData.subTabs, { merryGoRoundColors: true });
+
+        return (
+            <div className="matrix__presentation">
+                <Spinner isSpinnerActive={this.state.isSpinnerActive} />
+                <div className={`matrix__label matrix__label--horz${!scrolledRight ? ' horz-scroll' : ''}`}>
+                    <span>biosample</span>
+                    {svgIcon('largeArrow')}
+                </div>
+                <div className="matrix__presentation-content">
+                    <div className="matrix__label matrix__label--vert"><div>{svgIcon('largeArrow')}{context.matrix.y.label}</div></div>
+                    {showOrganismRequest ? <GetTargets /> : null }
+                    <TargetTabPanel tabs={headers} selectedTab={this.state.selectedTab} tabPanelCss="matrix__data-wrapper">
+                        <TargetTabPanel tabs={this.subTabsHeaders} selectedTab={selectedSubTab} tabPanelCss="matrix__data-wrapper" handleTabClick={this.subTabClicked} fontColors={fontColors}>
+                            {targetData && targetData.headerRow && targetData.headerRow.length !== 0 && targetData.dataRow && targetData.dataRow.length !== 0 ?
+                                  <div className="matrix__data" onScroll={this.handleOnScroll} ref={(element) => { this.scrollElement = element; }}>
+                                      <DataTable tableData={convertTargetDataToDataTable(targetData, selectedSubTab)} />
+                                  </div>
+                              :
+                                  <div className="matrix__warning">
+                                      <br />
+                                      { targetData && Object.keys(targetData).length === 0 ? 'Select an organism to view data.' : 'No data to display.' }
+                                      <br /><br /><br />
+                                  </div>
+                            }
+                            <br />
+                        </TargetTabPanel>
+                        <br />
```

Comment:
What are the two `<br />`s for? I don’t even see them in the generated HTML.

Reply:
I can show you. Go to [https://encd-4917-21525e593-phil.demo.encodedcc.org/target-matrix/?type=Experimen](https://encd-4917-21525e593-phil.demo.encodedcc.org/target-matrix/?type=Experimen)t. If you look "Select an Organism to view data", you will see the break points.

Reply:
OK, I see them now. I deleted both `<br>` from the DOM and didn’t see any difference though. I’m not sure what they do here.

Reply:
Move spacing to SASS.